pkgs/applications/audio/ardour/default.nix

```diff
@@ -111,35 +107,43 @@ in stdenv.mkDerivation rec {
     taglib
     vamp-plugin-sdk
     libarchive
+    libpulseaudio
+    libuv
+    libwebsockets
+    cppunit
+    readline
   ];
 
   wafConfigureFlags = [
     "--optimize"
     "--docs"
     "--use-external-libs"
     "--freedesktop"
-    "--with-backends=jack,alsa,dummy"
+    "--with-backends=jack,alsa,dummy,pulseaudio"
+    "--qm-dsp-include=${qm-dsp}/include/qm-dsp"
   ];
 
-  NIX_CFLAGS_COMPILE = "-I${qm-dsp}/include/qm-dsp";
+  patches = [
```

Comment:
We prefer having patches just after `src`, see https://discourse.nixos.org/t/document-attribute-ordering-in-package-expressions/4887.

Reply:
Addressed in 84ca1c92539738c904b1ca35f264f2f02a370b56